api.py

```diff
@@ -121,18 +129,46 @@ def move_coarse_focus(degrees: float, err_tol: float = 0.01) -> None:
 
     _send_motor_control_packet(motor_axes["focus_coarse"], degrees)
 
-def _send_motor_control_packet(motor_axis: int, degrees: float) -> None:
+def _send_motor_control_packet(motor_axis: bytes, degrees: float) -> None:
     """Send a motor control packet.
     
     :param motor_axis: the index of the motor the packet is commanding.
     :param degrees: the number of degrees to move that axis.
     :returns: None
 
     An internal implementation function which directly sends motor control
-    packets to the stepper controller. This does not do error checking or
-    axis translation and should only be used to implement the above API functions.
+    packets to the stepper controller. The packet size is 1 byte.
+    This does not do error checking or axis translation and should only be 
+    used to implement the above API functions.
     """
-    raise NotImplementedError()
+
+    if degrees > 1:
+        direction_packet = 1
+    else:
+        direction_packet = 0
+
+    steps_full = int(abs(degrees)//step_degrees) #round down to integer
+
+    while steps_full > 255:
+        steps_full -= 255
+        ser.write(motor_axis)
+        ser.write(direction_packet.to_bytes(1, byteorder = "big"))
+        ser.write(int(255).to_bytes(1, byteorder = "big"))
```

Comment:
Would you prefer if we just used a big enough int (int32? int64?) for the protocol that didn't need to be split? If this works it's probably worth not messing with it though.

Reply:
200 is a full rotation on the stepper so I was thinking this would never happen, I just included it to cover the edge case.

Reply:
ah cool, sounds good then.